hass_nabucasa/remote.py

```diff
@@ -172,8 +172,13 @@ def certificate(self) -> Optional[Certificate]:
                     "Home Assistant Cloud",
                     const.MESSAGE_REMOTE_READY,
                 )
+
         await self._acme.hardening_files()
 
+        # aiohttp_runner comes available when Home Assistant has started.
+        while self.cloud.client.aiohttp_runner is None:
+            await asyncio.sleep(1)
+
```

Comment:
Do we not wait on cloud integration until we are connected? Run this as task directly on loop and don't prevent the startup of core?

Reply:
This requires the HTTP integration to respond to the Home Assistant Start event. We startup before.

Reply:
This task is already run in the background, see a couple of lines before -> create_task.